lighthouse-core/audits/font-display.js

```diff
@@ -25,11 +25,13 @@ const UIStrings = {
     'webfonts are loading. ' +
     '[Learn more](https://web.dev/font-display/).',
   /**
-   * @description A warning message that is shown when Lighthouse couldn't automatically check some of the page's fonts and that the user will need to manually check it.
-   * @example {https://font.cdn.com/} fontURL
+   * @description [ICU Syntax] A warning message that is shown when Lighthouse couldn't automatically check some of the page's fonts, telling the user that they will need to manually check them.
+   * @example {https://font.cdn.com/} fontOrigin
    */
-  undeclaredFontURLWarning: 'Lighthouse was unable to automatically check the font-display value ' +
-    'for the following URL: {fontURL}.',
+  undeclaredFontOriginWarning:
+    'Lighthouse was unable to automatically check the {fontCountForOrigin, plural, ' +
```

Comment:
This is illegal. Can't have text outside a complex ICU message. It should be moved to be repeated in each option.

go/icu-message-migration#message-should-not-have-text-including-whitespaces-andor-placeholders-preceding-or-following-icu-complex-syntax

Reply:
> This is illegal. Can't have text outside a complex ICU message. It should be moved to be repeated in each option.

arg, makes sense (for those without access, other languages might need to change other parts outside of the brackets to grammatically work with the plural).

Seems like something we could automatically reject in the future.